docs/docs/guides/authentication/cloudentity.md

```diff
@@ -0,0 +1,110 @@
+# Cloudentity
+
+Kusk Gateway enables using third-party authentication and authorization management solutions such as Cloudentity.
+
+Developers can utilize Cloudentity's platform to control their authorization flows while still using OpenAPI with Kusk to configure gateway level rules.
+
+This separation of concerns is important when your authorization flow complexity grows or the number of services talking to each other require more authorization granularity.
+
+## Setup guide
+
+By the end of this guide, you will have deployed an API to Kusk and used Cloudentity to block requests to your API. 
+
+### 1. Create Kusk Authorizer in Cloudentity
+
+On Cloudentity's platform, go to **Enforcement > Authorizers**, click **CREATE GATEWAY**, and from the list that's displayed choose Kusk.
+
+![cloudentity authorizers list](./img/cloudentity-authorizers.png)
+
+### 2. Install Cloudentity's authorizer in your cluster
```

Comment:
Do you want to do uninstall section as well? The command would then be `helm uninstall cloudentity-authorizer --namespace kusk-system`.

Reply:
Maybe we could have it in another page, like a reference page instead of in the guide.